src/main.rs

```diff
@@ -17,9 +17,13 @@ use config::{CargoCratesToml, Package, UserConfig};
 
 /// Wrap the desired main and display errors in a fashion consistent with the
 /// rest of the messages.
-fn main() {
-    if let Err(err) = wrapped_main() {
-        error!("{}", err);
+fn main() -> std::process::ExitCode {
+    match wrapped_main() {
+        Ok(_) => std::process::ExitCode::SUCCESS,
+        Err(err) => {
+            error!("{}", err);
+            std::process::ExitCode::FAILURE
```

Comment:
Like I said in https://github.com/PaulDance/cargo-liner/pull/9#discussion_r1442174198, I would prefer if `ExitCode` was imported instead of being used with the verbose `std::process::ExitCode`. Add a `use std::process::ExitCode;` where appropriate at the top of the file, replace these `std::process::ExitCode` with `ExitCode` and it should be good.

Reply:
Sorry, I missed that

Reply:
It's fine, don't worry, that's what review are for!